specification/web/resource-manager/Microsoft.Web/2016-03-01/ResourceProvider.json

```diff
@@ -307,6 +307,51 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/verifyHostingEnvironmentVnet": {
+      "post": {
+        "summary": "Verifies if this VNET is compatible with an App Service Environment.",
+        "description": "Verifies if this VNET is compatible with an App Service Environment.",
+        "operationId": "VerifyHostingEnvironmentVnet",
+        "consumes": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml",
+          "application/x-www-form-urlencoded"
+        ],
+        "produces": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml"
+        ],
```

Comment:
I'll have to check with @naveedaz, not sure where this work item might be tracked. Navy, do you have a work item tracked for this?